deployment/get_bins

```diff
@@ -38,6 +38,13 @@ LATEST_URL=http://lna-ci-slave2.kaltura.com/bin/latest/${OS}/release
 
 function download_files()
 {
+    [ -w ${LIVE_ROOT_PATH} ] && DST_DIR=${LIVE_ROOT_PATH} || DST_DIR=${VERSION}
+
+	mkdir -p "${DST_DIR}/bin"
+    mkdir -p "${DST_DIR}/liveRecorder/bin"
+
+    echo "current dir: [`pwd`]"
```

Comment:
Please add comments as to what you are doing for ignorants like me who don't understand all the syntax